spec/Entity/ShopBillingDataSpec.php

```diff
@@ -18,48 +18,49 @@
 
 final class ShopBillingDataSpec extends ObjectBehavior
 {
-    public function let(): void
-    {
-        $this->beConstructedWith('Needful Things', '000222', 'US', 'Main St. 123', 'Los Angeles', '90001');
-    }
-
     public function it_implements_shop_billing_data_interface(): void
     {
         $this->shouldImplement(ShopBillingDataInterface::class);
     }
 
     public function it_has_no_id_by_default(): void
     {
-        $this->id()->shouldReturn(null);
+        $this->getId()->shouldReturn(null);
     }
 
     public function it_has_company(): void
```

Comment:
This is not a blocker for this PR but I would encourage you to add `a` or `an` articles before param.
F.e: `it_has_company` -> `it_has_a_company`

Reply:
yeah, right